pkg/helpers/motd/motd.go

```diff
@@ -0,0 +1,43 @@
+package motd
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog"
+)
+
+// Display the necessary MOTD if it's configured in the system.
+// Note that the MOTD comes from the "motd" configMap, which
+// should be in the "openshift-config" namespace. This needs to be configured
+// by the deployer.
+func DisplayMOTD(coreClient corev1client.CoreV1Interface, out io.Writer) error {
+	motdcm, err := coreClient.ConfigMaps("openshift-config").Get("motd", metav1.GetOptions{})
```

Comment:
`openshift-config` might not be a good place, since it's meant more for configuring operators and cluster. I'd suggest moving this to `openshift` since that's always been a place readable by every user of the cluster. /cc @smarterclayton

Reply:
Thanks for the feedback!